src/main/java/study/racing/view/ResultView.java

```diff
@@ -0,0 +1,32 @@
+package study.racing.view;
+
+import study.racing.domain.Car;
+import study.racing.domain.RacingRecord;
+
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class ResultView {
+
+    public static void printResult(RacingRecord racingRecord) {
+        Set<Car> cars = racingRecord.getRecords().keySet();
+
+        for (int i = 1; i <= racingRecord.getTotalRound(); i++) {
+            int round = i;
+            cars.forEach(car -> {
+                int distance = racingRecord.getRecords().get(car).get(round);
```

Comment:
```suggestion
                int distance = racingRecord.getRecords()
                                                               .get(car)
                                                               .get(round);
```
한줄에 하나의 점 `.` 을 찍으면 가독성이 더 좋을거 같아요.

Reply:
넵넵 ㅎ감사합니다